src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -45,6 +45,7 @@ def self.attributes
           { name: :keep_unknown_lv },
           { name: :lvm2 },
           { name: :is_lvm_vg },
+          { name: :lvm_group },
```

Comment:
According to AutoYaST documentation, there is no `lvm_group` attribute in a `<drive>` section. Only `<partition>` sections have that.

Reply:
BTW, `pesize` is in `drive` and not in `partition` where it would logically belong to.

Reply:
The autoyast profile looks not very consistent here and we should try to make it more logical going forward.

Reply:
If it would be only `lvm_group` and `pesize`... There are quote some attributes in the current specification that are UTTERLY misplaced.

But in my opinion if we want to introduce another way of specifying the usage of full disks, we must first:

- decide/design how it's going to look like
- implement it
- then, document it

And that includes thinking about import, export, etc. and thinking about full disks as PV, full formatted disks, etc.

Just adding another attribute to the `<drive>` section "since we are here" sounds like the recipe to repeat the problem in the future (discovering via bug report than the code supported yet another way of doing things because it sounded like a good idea to whoever was touching the code). :wink: